shacl12-core/index.html

```diff
@@ -1654,20 +1654,21 @@ <h3>Node Expressions</h3>
 					One of these parameters can be the <dfn>key parameter</dfn> that uniquely identifies the <a>function name</a>.
```

Comment:
I read "TEXTUAL DEFINITION" as implying:

> Each node expression function has an IRI as its function name.
> Node expression functions can declare one or more node expression parameters.
> Each of these parameters has an IRI. 

reads as if the only way to invoke a function is by its function name which isn't true (IRI expression, literal expression)

Alternative:

A node expression is one of
1. A function called by its function name
2. An IRI expression
3. A literal expression

and the definitions of "IRI expression" and "literal expression" need changing.

Reply:
This question is not directly related to this PR. This definition was there before. It wouldn't surprise anyone if the details of node expressions will be refined in the next couple of iterations. I suggest if you think this block needs changing, you could raise a fresh issue and maybe make a specific PR, because I am not exactly sure what you mean. The relationship between the function IRI and the concept of the key parameter is also unclear to me. It will become clearer once we start on the Node Expr document (which may make sense pretty soon).

For housekeeping I'll mark this question resolved - please reopen if you disagree.

Reply:
> This question is not directly related to this PR.

The previous update as pushed through to get into FPWD.